cmd/soroban-rpc/internal/config/config_option.go

```diff
@@ -101,7 +99,7 @@ func (o *ConfigOption) setValue(i interface{}) (err error) {
 		}
 	}()
 	parser := func(option *ConfigOption, i interface{}) error {
-		panic(fmt.Sprintf("no parser for flag %s", o.Name))
+		return errors.Errorf("no parser for flag %s", o.Name)
```

Comment:
can you add an assertion in config_option_test.go that demonstrates the expected outcome if this gets triggered?